tests/__snapshots__/typescript-mock-data.spec.ts.snap

```diff
@@ -33,6 +49,22 @@ export const anAvatar = (overrides?: Partial<Avatar>): Avatar => {
     };
 };
 
+export const aMutation = (overrides?: Partial<Mutation>): Mutation => {
+    return {
+        updateUser: aUser(),
+        ...overrides
+    };
+};
```

Comment:
I don't think we should generate mock data for mutations.
We already exclude `Query` type
See here

Reply:
Yeah, makes sense
Fixed